.travis.yml

```diff
@@ -86,15 +130,15 @@ script:
      fi
    # Print the compiler version
    - $MYCXX -v
-   # Build the code and do a quick check (debug mode) or a full tests run (non-debug mode)
-   - if [ $DEBUG == "NO" ]; then
-        export MFEM_TEST_TARGET="test";
-     else
-        export MFEM_TEST_TARGET="check";
-     fi
+        
    # Build and check/test MFEM, its examples and miniapps
    - cd $TRAVIS_BUILD_DIR &&
```

Comment:
Added code coverage option and ability to specify different number for mpi tasks in make 'test'.